src/pages/MoviesList/MovieApp.jsx

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable prefer-const */
```

Comment:
We normally try to avoid eslint-disable rules as much as possible, in this case, this is not needed, you can omit it.